Broken in nixpkgs

```
$ nix-shell -p haskellPackages.jenkinsPlugins2nix
error: Package ‘jenkinsPlugins2nix-0.5.0.0’ in /nix/store/al476r5q8bifinbrysh37yzm7mbryxvd-nixpkgs/nixpkgs/pkgs/development/haskell-modules/hackage-packages.nix:165065 is marked as broken, refusing to evaluate.

       a) To temporarily allow broken packages, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_BROKEN=1

        Note: For `nix shell`, `nix build`, `nix develop` or any other Nix 2.4+
        (Flake) command, `--impure` must be passed in order to read this
        environment variable.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowBroken = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowBroken = true; }
       to ~/.config/nixpkgs/config.nix.
(use '--show-trace' to show detailed location information)
```

This is due to the upper bounds on `attoparsec` and `tasty`. Trying to build with `NIXPKGS_ALLOW_BROKEN=1` results in

```
Configuring jenkinsPlugins2nix-0.5.0.0...

Setup: Encountered missing or private dependencies:
attoparsec >=0.13 && <0.14, tasty-hspec >=1.1 && <1.2
```

However, just bumping the upper bound is not enough, as it results in another error in the test suite:

```
test/Nix/JenkinsPlugins2Nix/Tests/Parser.hs:16:9: error:
    Not in scope: type constructor or class ‘Hspec.Spec’
    Module ‘Test.Tasty.Hspec’ does not export ‘Spec’.
   |
16 | spec :: Hspec.Spec
   |         ^^^^^^^^^^

test/Nix/JenkinsPlugins2Nix/Tests/Parser.hs:17:8: error:
    Not in scope: ‘Hspec.describe’
    Module ‘Test.Tasty.Hspec’ does not export ‘describe’.
   |
17 | spec = Hspec.describe "parser spec" $ do
   |        ^^^^^^^^^^^^^^

test/Nix/JenkinsPlugins2Nix/Tests/Parser.hs:18:3: error:
    Not in scope: ‘Hspec.it’
    Module ‘Test.Tasty.Hspec’ does not export ‘it’.
   |
18 |   Hspec.it "parser github-pullrequest manifest" githubPr
   |   ^^^^^^^^

test/Nix/JenkinsPlugins2Nix/Tests/Parser.hs:20:13: error:
    Not in scope: type constructor or class ‘Hspec.Expectation’
    Module ‘Test.Tasty.Hspec’ does not export ‘Expectation’.
   |
20 | githubPr :: Hspec.Expectation
   |             ^^^^^^^^^^^^^^^^^

test/Nix/JenkinsPlugins2Nix/Tests/Parser.hs:21:27: error:
    Not in scope: ‘Hspec.shouldBe’
    Module ‘Test.Tasty.Hspec’ does not export ‘shouldBe’.
   |
21 | githubPr = Right expected `Hspec.shouldBe` runParseManifest githubPrManifestText
   |                           ^^^^^^^^^^^^^^^^
```

--- a/flake.nix
+++ b/flake.nix
@@ -2,7 +2,7 @@
   description = "jenkinsPlugins2nix";
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
-    nixpkgs.url = "github:NixOS/nixpkgs/21.05";
+    nixpkgs.url = "github:NixOS/nixpkgs/22.11";
     flake-compat = {
       url = "github:edolstra/flake-compat";
       flake = false;

--- a/jenkinsPlugins2nix.cabal
+++ b/jenkinsPlugins2nix.cabal
@@ -1,5 +1,5 @@
 name:                jenkinsPlugins2nix
-version:             0.5.0.0
+version:             0.5.0.1
 synopsis:            Generate nix for Jenkins plugins.
 description:         Generate nix for Jenkins plugins.
 homepage:            https://github.com/Fuuzetsu/jenkinsPlugins2nix#readme
@@ -20,7 +20,7 @@ library
                        Nix.JenkinsPlugins2Nix.Parser
                        Nix.JenkinsPlugins2Nix.Types
   build-depends:       base           >=4.7  && < 5
-                     , attoparsec     >=0.13 && <0.14
+                     , attoparsec     >=0.13 && <0.15
                      , bytestring     >=0.10 && <0.11
                      , containers     >=0.6  && <0.7
                      , cryptonite     >=0.24 && <0.30
@@ -53,7 +53,8 @@ test-suite jenkinsPlugins2nix-test
   build-depends:       base
                      , containers
                      , jenkinsPlugins2nix
-                     , tasty-hspec         >=1.1 && <1.2
+                     , hspec
+                     , tasty-hspec         >=1.1 && <1.3
                      , text
   ghc-options:         -threaded -rtsopts -with-rtsopts=-N -Wall
   default-language:    Haskell2010

--- a/test/Nix/JenkinsPlugins2Nix/Tests/Parser.hs
+++ b/test/Nix/JenkinsPlugins2Nix/Tests/Parser.hs
@@ -11,6 +11,7 @@ import qualified Data.Set as Set
 import qualified Data.Text as Text
 import           Nix.JenkinsPlugins2Nix.Parser (runParseManifest)
 import           Nix.JenkinsPlugins2Nix.Types
+import qualified Test.Hspec as Hspec
 import qualified Test.Tasty.Hspec as Hspec
 
 spec :: Hspec.Spec

--- a/test/Spec.hs
+++ b/test/Spec.hs
@@ -1,7 +1,7 @@
 module Main where
 
 import qualified Nix.JenkinsPlugins2Nix.Tests.Parser as Parser
-import qualified Test.Tasty.Hspec as Hspec
+import qualified Test.Hspec as Hspec
 
 main :: IO ()
 main = Hspec.hspec $ do

